.github/workflows/gradle.yml

```diff
@@ -76,12 +77,7 @@ jobs:
         if: success() && github.ref == 'refs/heads/master'
         run: ./gradlew --no-daemon integrationTest --scan
 
-      - name: Run Standard Source Tests (PR)
```

Comment:
i want to make sure that the tests run properly. so made this change so the PR build will run the standard tests for this integration. once i am sure it is working i will revert this out.